dGame/dComponents/PetComponent.h

```diff
@@ -103,7 +98,7 @@ class PetComponent : public Component
 	 * @param typeId extra information about the command, e.g. the emote to play
 	 * @param overrideObey unused
 	 */
-	void Command(NiPoint3 position, LWOOBJID source, int32_t commandType, int32_t typeId, bool overrideObey);
+	void Command(const NiPoint3& position, const LWOOBJID source, const int32_t commandType, const int32_t typeId, const bool overrideObey);
```

Comment:
I believe it is still good to declare a variable const so that you cant accidentally modify it in your function body.  It does prevent you from accidentally changing the value mindlessly

Reply:
If you're consistent with it you can also tell when a function is going to modify an input because said input will not be declared const